cpp/arcticdb/async/python_bindings.cpp

```diff
@@ -16,15 +16,16 @@ namespace arcticdb::async {
 void register_bindings(py::module &m) {
     auto async = m.def_submodule("cpp_async", "Asynchronous processing");
 
-    py::class_<TaskScheduler, std::shared_ptr<TaskScheduler>>(async, "TaskScheduler")
```

Comment:
The original line is a mere 85 cols long. Can you set the line width higher, e.g. to 120?